src/features/apiDelegation/components/SummaryPage/SummaryPage.tsx

```diff
@@ -214,7 +214,7 @@ export const SummaryPage = ({
                 nextPath={
                   '/' + ApiDelegationPath.OfferedApiDelegations + '/' + ApiDelegationPath.Receipt
                 }
-                nextText={t('api_delegation.confirm_delegation')}
+                nextText={t('common.confirm')}
```

Comment:
```suggestion
                nextText={t('common.confirm_delegation')}
```
Or make a new text key `confirm`

Reply:
That's exactly what I did here? 😅

Reply:
No, you changed it to common.confirm, but there is no text key with that name 😅
So you either have to make that key or you have to switch to a key that exists.
![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/f887e808-2abe-4c89-87ec-b673612ec43b)

Reply:
Ahh f... this task is killing me, too many small changes

Reply:
I feel ya... 🫠